target/linux/realtek/dts-5.15/rtl838x_hpe_1920.dtsi

```diff
@@ -5,7 +5,8 @@
 
 / {
 	chosen {
-		bootargs = "console=ttyS0,38400";
+		bootargs = "earlycon";
```

Comment:
No need to duplicate 'bootargs' here.

Reply:
duplicate from where? you mean from the rtl838x.dtsi? Fair enough.